data/json/items/gun/36paper.json

```diff
@@ -4,8 +4,16 @@
     "copy-from": "pistol_revolver_cap_ball",
     "looks_like": "sw_619",
     "type": "GUN",
-    "name": { "str_sp": "Colt M1861 Navy" },
+    "name": { "str_sp": "civil war six-shooter" },
```

Comment:
Looks like you forgot to replace `str_sp` with `str` for generic names, otherwise, the plural form of this name will be the same as the singular:
```
msgid ".45/.410 shotshell revolver"
msgid_plural ".45/.410 shotshell revolver"
```